examples/quickstart/20-pod.yaml

```diff
@@ -55,9 +53,6 @@ spec:
         name: sonobuoy-plugins-cm
     # NOTE: This will drop the results on the hostpath of the machine as an example.
     # We recommend using a persistent volume for more permanent recording.
-    - name: output-volume
-      hostPath:
```

Comment:
Changed it to `emptyDir`, @abiogenesis-now please take a look at docs/configuration.md and let me know if you have any changes.